# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/interfaces/nsIAccessibleVirtualCursorChangeEvent.idl
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/interfaces/nsIAccessibleVirtualCursorChangeEvent.idl
--- a/accessible/interfaces/nsIAccessibleVirtualCursorChangeEvent.idl	Wed Nov 29 03:53:20 2023 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,29 +0,0 @@
-/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*-
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "nsIAccessibleEvent.idl"
-
-/*
- * An interface for virtual cursor changed events.
- * Passes previous cursor position and text offsets.
- */
-[scriptable, builtinclass, uuid(a58693b1-009e-4cc9-ae93-9c7d8f85cfdf)]
-interface nsIAccessibleVirtualCursorChangeEvent : nsIAccessibleEvent
-{
-  /**
-   * Previous object pointed at by virtual cursor, null if none.
-   */
-  readonly attribute nsIAccessible oldAccessible;
-
-  /**
-   * New object pointed at by virtual cursor, null if none.
-   */
-  readonly attribute nsIAccessible newAccessible;
-
-  /**
-   * Reason for virtual cursor move.
-   */
-  readonly attribute short reason;
-};